.github/CODEOWNERS

```diff
@@ -13,3 +13,7 @@ internal/connector/**/*  @valdar @dhirajsb
 # common
 pkg/**/*  @pb82 @davidffrench @machi1990 @akoserwal @pawelpaszki @JameelB @miguelsorianod @ziccardi @valdar @dhirajsb
 test/**/* @pb82 @davidffrench @machi1990 @akoserwal @pawelpaszki @JameelB @miguelsorianod @ziccardi @valdar @dhirajsb
+
+# api
+openapi/ @bf2fc6cc711aee1a0c2a/devexp-sdks
```

Comment:
Github is complaining with `Unknown owner on line 18: make sure the team @bf2fc6cc711aee1a0c2a/devexp-sdks exists and has write access to the repository`
is that to be expected?